vue/src/components/BodyTextComponent.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+
+// Define props for the component
+const props = defineProps<{
+  size?: string;
+  width?: string; // doesn't work
+}>();
+</script>
+
+<template>
+  <p :class="[
+      'text-[#4A4A4A]',
+      'font-inter',
+      'p-4',
+      size ? `text-${size}` : 'text-base sm:text-xl',
+      width ? `font-${width}` : 'font-normal'
+    ]">
+    <slot />
```

Comment:
så bliver teksten skrevet der og og givet videre som prop -- så det kan være fra json filer eller bare en string?
i think it work sthat way
```suggestion
    {{ textContent }}
```